.travis.yml

```diff
@@ -59,6 +59,11 @@ jobs:
     - env: EMBER_CLI_BROCCOLI_2=false
     - env: EMBER_CLI_SYSTEM_TEMP=false
 
+    - env: NAME=NODE_10_NPM
+      node_js: 10
+      install: npm install
+      script: npm run test:all
```

Comment:
this sounds like it would add significantly more time to our already long CI pipeline 😢

Reply:
I guess it depends on what you mean by "significant" 😂 

In the case of individual jobs it seems like the npm run is consistently faster than yarn in the 2 out of 2 builds that I've seen now 😂 Not a great sample size but considering that's without a package-lock.json I'm pretty impressed.

Secondly, it seems like we are getting a 5 job concurrency on the big stage and adding this job brings the total number of jobs to 10. In reality I guess that means that as long as we keep the job count below 10 we should be fine when it comes to elapsed time 🤔